README.md

```diff
@@ -37,6 +37,7 @@
 * [urlr.me](https://urlr.me/en) - Reliable URL shortener that provide an API
 * [v.gd](https://v.gd)
 * [vo.la](https://vo.la/) - Korean URL Shortener Service
+* [spoo.me](https://spoo.me/)
```

Comment:
Sorry, but I would appreciate it if you could sort them alphabetically.

Reply:
Updated!
Additionally, it would be great if you could include a note in the contributing.md file about sorting services alphabetically
Cheers